Yona/Yona/UserRequestManager.swift

```diff
@@ -141,6 +143,27 @@ class UserRequestManager{
         }
     }
     
+    func getUserFetchURL() -> String {
+        return compareAndSwapSchemes(leftURL: getSavedUserFromUserDefaults().getSelfLink!, rightURL: EnvironmentManager.baseUrlString()!,userLinkFromKeychain: KeychainManager.sharedInstance.getUserSelfLink()!)
```

Comment:
Now this can become:
```
        return compareAndSwapSchemes(userURLStr: getSavedUserFromUserDefaults().getSelfLink!, environmentBaseURLStr: EnvironmentManager.baseUrlString()!,storedUserUrlStr: storedUserUrl)
```
(see also the request below to rename the parameters if this function)

Reply:
Done.